packages/demo/src/plasma/navigation/SideNavigation.tsx

```diff
@@ -42,7 +42,8 @@ const CollapsibleSideSection: React.FC<{title: string}> = ({title, children}) =>
             expandable
             expanded={expanded}
             onClick={() => setExpanded(!expanded)}
-            title={<NavLink label={title} />}
+            // isActive={false} stops navbar loading near bottom
```

Comment:
is this useful to keep in the code? :thinking:

Reply:
Yes and no - I found it interesting that it was because `isActive` was accidentally set that the nav kept loading at the bottom, but I know its hardly critical information. I don't mind removing